{{ cookiecutter.format }}/{{ cookiecutter.class_name }}/main.m

```diff
@@ -82,7 +86,7 @@ int main(int argc, char *argv[]) {
         }
 
         // Set the home for the Python interpreter
-        python_home = [NSString stringWithFormat:@"%@/support/python-stdlib", resourcePath, nil];
+        python_home = [NSString stringWithFormat:@"%@/Python.framework/Versions/{{ cookiecutter.python_version|py_tag }}", frameworksPath, nil];
```

Comment:
`Python.framework/Versions/{{ cookiecutter.python_version|py_tag }}` appears 3 times in this function, and `{{ cookiecutter.python_version|py_tag }}` appears 5 times. Can we factor them out into variables?